### PR TITLE
Feature/card styling

### PR DESCRIPTION
**Pull request by stepris**

### CODE REVIEW

File: css/css-structure/taco-truck/components/header.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.header {
+  background-color: var(--header-bg);
+  color: var(--header-text);
+  align-self: flex-start;
```
Comment (MoritzBhl) [inline css/css-structure/taco-truck/components/header.css:4]: nice alignment

File: css/css-structure/taco-truck/components/card.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.card {
+  /* background-color: rgb(46, 179, 46); */
+  /* color: rgb(245, 224, 224); */
+  background-color: var(--card-bg);
+  color: var(--card-text);
```
Comment (MoritzBhl) [inline css/css-structure/taco-truck/components/card.css:5]: i like this color

File: css/css-structure/taco-truck/components/main.css
Comment (MoritzBhl) [inline css/css-structure/taco-truck/components/main.css:1]: I like the way you aligned it with padding and align-self. Nicely done 👍

File: css/css-structure/taco-truck/index.html
Comment (MoritzBhl) [inline css/css-structure/taco-truck/index.html:1]: The classes are very well structured

**Review by MoritzBhl — APPROVED** [review]
bg,f8f8f8-flat,750x,075,f-pad,750x1000,f8f8f8

**Comment by stepris** [comment]
Ready to merge, thanks!

Let's go 💪

**Comment by stepris** [comment]
Oh, I accidentally closed it

**Comment by stepris** [comment]
Good job everyone, fantastic feature 😄

**Pull request merged**